connector_dns/models/dns.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Elico Corp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import models, fields, api
+
+
+class DNSBackend(models.Model):
+    _name = 'dns.backend'
+    _inherit = 'connector.backend'
+    _backend_type = 'dns'
+
+    def _select_version(self):
+        return []
+
+    login = fields.Char(
+        string='Login',
+        help="Provider's login.",
+        required=True
+    )
+    password = fields.Char(
+        string='Password',
+        help="Provider's password.",
+        required=True
+    )
+    state = fields.Selection(
```

Comment:
Is this being used? I can't seem to find any sort of corresponding logic here? Did this instead get implemented as `state` in the abstract binding?